pq-crypto/pq_kem.c

```diff
@@ -0,0 +1,114 @@
+// -----------------------------------------------------------------------------
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// KEM Library
+// Abstract: Post-Quantum Algorithm API
+// -----------------------------------------------------------------------------
+
+#include "pq_kem.h"
+#include "pq_kem_params_size.h"
+#include "sike_r3/P434/P434_api.h"
+#include "openssl/mem.h"
+
+EVP_PQ_KEM_CTX *EVP_PQ_KEM_CTX_new(void) {
+  return OPENSSL_malloc(sizeof(EVP_PQ_KEM_CTX));
+}
+
+int EVP_PQ_KEM_CTX_init(EVP_PQ_KEM_CTX *kem_ctx, const EVP_PQ_KEM *kem) {
+
+  if (kem_ctx == NULL || kem == NULL) {
+    return 0;
+  }
+
+  kem_ctx->kem = kem;
+
+  kem_ctx->public_key    = OPENSSL_malloc(kem->public_key_length);
```

Comment:
Dunno what is going on here with spacing, but it looks funny :)

Reply:
I aligned the equal signs, looks nicer to me that way :) if we don't do that in AWS-LC I'll remove the extra space?

Reply:
lol - I'm really indifferent